packages/http-date/http-date.0.1/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "HTTP Datetime encoder/decoder"
+description: "RFC 9110 compliant HTTP datetime decoder and encoder"
+maintainer: ["Bikal Lem"]
+authors: ["Bikal Lem"]
+license: "MPL-2.0"
+homepage: "https://github.com/bikallem/http-date"
+bug-reports: "https://github.com/bikallem/http-date/issues"
+depends: [
+  "ocaml" {>= "5.0"}
+  "dune" {>= "3.6" & >= "2.6"}
+  "ptime" {>= "1.1.0"}
+  "menhir" {>= "20220210" & with-build}
```

Comment:
```suggestion
  "menhir" {>= "20220210" & build}
```

I think that's the syntax. That's what https://opam.ocaml.org/doc/Manual.html#Filters suggests.